Use "region" to disambiguate "location"

andynava opened the pull request:
It's a big ambiguous and inconsistent how "location" and "region" are used interchangeably.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

andynava committed:
Use "region" to disambiguate "location"

It's a big ambiguous and inconsistent how "location" and "region" are used interchangeably.
